pythonx/jedi_vim.py

```diff
@@ -1074,11 +1081,11 @@ def py_import_completions():
 
 
 @catch_and_print_exceptions
-def new_buffer(path, options='', using_tagstack=False):
+def new_buffer(path, options='', using_tagstack=False, force_buffer=False):
     # options are what you can to edit the edit options
-    if int(vim_eval('g:jedi#use_tabs_not_buffers')) == 1:
+    if int(vim_eval('g:jedi#use_tabs_not_buffers')) == 1 and not force_buffer:
```

Comment:
For tabs the old behavior is fine IMO. I use it regularly.

Reply:
You're right, will fix this :+1: